test/serializers/read_attribute_for_serialization_test.rb

```diff
@@ -7,20 +7,28 @@ class ReadAttributeForSerializationTest < ActiveSupport::TestCase
       class Parent < ActiveModelSerializers::Model
         attr_accessor :id
       end
+
       class Child < Parent
         attr_accessor :name
       end
+
       class ParentSerializer < ActiveModel::Serializer
         attributes :$id
 
         define_method(:$id) do
           object.id
         end
       end
+
       class ChildSerializer < ParentSerializer
         attributes :name
       end
 
+      def test_for_serializer_initialized_with_nil_object
+        child = ChildSerializer.new(nil)
+        assert_equal nil, child.read_attribute_for_serialization(:name)
```

Comment:
Agreed. But it gives me a trace from the gem instead of my app. So maybe in the initializer, we could raise an exception if somebody attempts to initialize with a nil because the code expects the object passed in to be not nil. I didn't want to mess with ```def initialize``` but this would be an additional check, maybe?